css/fonts.css

```diff
@@ -1,43 +1,39 @@
 @font-face {
   font-family: "Montserrat";
   src:
-    url("../fonts/MontserratAlternates-Bold.eot"),
-    url("../fonts/MontserratAlternates-Bold.eot?#iefix") format("embedded-opentype"),
-    url("../fonts/MontserratAlternates-Bold.woff") format("woff"),
-    url("../fonts/MontserratAlternates-Bold.woff2") format("woff2");
+    url("../fonts/Montserrat-Bold.eot"),
+    url("../fonts/Montserrat-Bold.eot?#iefix") format("embedded-opentype"),
+    url("../fonts/Montserrat-Bold.woff") format("woff"),
+    url("../fonts/Montserrat-Bold.woff2") format("woff2");
   font-weight: bold;
   font-style: normal;
 }
-
 @font-face {
   font-family: "Montserrat";
   src:
-    url("../fonts/MontserratAlternates-Medium.eot"),
-    url("../fonts/MontserratAlternates-Medium.eot?#iefix") format("embedded-opentype"),
-    url("../fonts/MontserratAlternates-Medium.woff") format("woff"),
-    url("../fonts/MontserratAlternates-Medium.woff2") format("woff2");
+    url("../fonts/Montserrat-Medium.eot"),
+    url("../fonts/Montserrat-Medium.eot?#iefix") format("embedded-opentype"),
+    url("../fonts/Montserrat-Medium.woff") format("woff"),
+    url("../fonts/Montserrat-Medium.woff2") format("woff2");
   font-weight: 500;
   font-style: normal;
 }
-
 @font-face {
   font-family: "Montserrat";
   src:
-    url("../fonts/MontserratAlternates-Regular.eot"),
-    url("../fonts/MontserratAlternates-Regular.eot?#iefix") format("embedded-opentype"),
-    url("../fonts/MontserratAlternates-Regular.woff") format("woff"),
-    url("../fonts/MontserratAlternates-Regular.woff2") format("woff2");
+    url("../fonts/Montserrat-Regular.eot"),
+    url("../fonts/Montserrat-Regular.eot?#iefix") format("embedded-opentype"),
+    url("../fonts/Montserrat-Regular.woff") format("woff"),
+    url("../fonts/Montserrat-Regular.woff2") format("woff2");
   font-weight: normal;
   font-style: normal;
 }
-
 @font-face {
   font-family: "Montserrat";
   src:
-    url("../fonts/MontserratAlternates-SemiBold.eot"),
-    url("../fonts/MontserratAlternates-SemiBold.eot?#iefix") format("embedded-opentype"),
-    url("../fonts/MontserratAlternates-SemiBold.woff") format("woff"),
-    url("../fonts/MontserratAlternates-SemiBold.woff2") format("woff2");
+    url("../fonts/Montserrat-SemiBold.eot"),
+    url("../fonts/Montserrat-SemiBold.eot?#iefix") format("embedded-opentype"),
+    url("../fonts/Montserrat-SemiBold.woff") format("woff"),
+    url("../fonts/Montserrat-SemiBold.woff2") format("woff2");
   font-weight: 600;
-  font-style: normal;
-}
+  font-style: normal; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)